bin/lokalise.js

```diff
@@ -0,0 +1,89 @@
+require('dotenv').config();
+
+const { LokaliseApi } = require('@lokalise/node-api');
+const fs = require('fs');
+const path = require('path');
+
+const lokaliseApi = new LokaliseApi({ apiKey: process.env.LOKALISE_API_TOKEN });
+
+const fetchPage = async(page) => {
+  const response = await lokaliseApi.keys.list({
+    'project_id': process.env.LOKALISE_PROJECT_ID,
+    page,
+    limit: 1000,
+    'include_translations': 1
+  });
+  return response.items;
+};
+
+const stringify = (thing, root = false, indent = 0) => {
+  let str = '';
+  if (root) {
+    str = str + 'export default ';
+  }
+
+  if (typeof thing === 'string') {
+    const quoted = thing.replace(/"/g, '\\"');
+    str = str + `"${quoted}"`;
+  } else {
+    str = str + '{\n';
+    const sorted = Object.keys(thing).sort((a, b) => a.localeCompare(b, undefined, { sensitivity: 'base' }));
+    for (const key of sorted) {
+      str = str + ' '.repeat(indent + 2) + `"${key}": `;
+      str = str + stringify(thing[key], false, indent + 2);
+      if (key !== sorted[sorted.length - 1]) {
+        str = str + ',';
+      }
+      str = str + '\n';
+    }
+    str = str + ' '.repeat(indent) + '}';
+  }
+
+  if (root) {
+    str = str + ';\n';
+  }
+
+  return str;
+};
+
+const run = async() => {
+  const i18n = {};
+  let items;
+  let page = 0;
+
+  while (!Array.isArray(items) || items.length > 0) {
```

Comment:
is items not an array only on the first run, or can it come back like that from the fetchPage request?

Reply:
It's not an array on the first run because it's just been initialised on line 51, but not set to anything yet.